rdsinstance/src/backup_lambda/cli.ts

```diff
@@ -0,0 +1,25 @@
+import { handler } from "./index";
+
+const run = async () => {
+  process.env.RDS_INSTANCE_IDENTIFIER = process.env.RDS_INSTANCE_IDENTIFIER ?? 'itd-pr-pdg-reportingdata';
```

Comment:
This seems odd - probably just left over from the original? If not, definitely don't think we should hardcode something like that in here

Reply:
Nah this is actually intentional. this is just a file for testing the handler locally in dry run mode

Reply:
I can remove the hardcoded identifier though and make it a CLI arg

Reply:
Nah that's fine!